app/models/api/openid_connect/id_token.rb

```diff
@@ -0,0 +1,52 @@
+module Api
+  module OpenidConnect
+    class IdToken < ActiveRecord::Base
+      belongs_to :authorization
+
+      before_validation :setup, on: :create
+
+      default_scope { where("expires_at >= ?", Time.zone.now.utc) }
```

Comment:
I'm pretty sure they fetch the still valid ones. This is equivalent to "expired_at time is greater than the current time" right? For example if the expired_at time is 5:30 and it is currently 5:15, then this token will still be valid. Perhaps it should be renamed?